gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/shared_policy_group/model/SharedPolicyGroup.java

```diff
@@ -90,6 +92,40 @@ public class SharedPolicyGroup {
      */
     private SharedPolicyGroupLifecycleState lifecycleState;
 
+    public io.gravitee.definition.model.v4.sharedpolicygroup.SharedPolicyGroup deploy() {
```

Comment:
🧌 but `toDeployedDefinition` (my code style 🙈 )

Reply:
it is not only about mapping here. we have a real mutation of the object so I do not agree with `to...`
maybe the solution is to have:
- a `deploy()` function only mutating the state
- a dedicated `toDefinition()` function only responsible of doing the transformation to a definition
wdyt ?

Reply:
That's fine with me. It's easy to understand with java type. 
--
For me, it's all about starting with one object and moving on to another, more specific to the business. 
From any spg Object to another Object the domain for deployment. the deploy action isn't over yet and need more step. 

But it's really style, code habit I think. like that it's good ;)